modules/javafx.graphics/src/main/java/javafx/geometry/Point3D.java

```diff
@@ -410,10 +410,15 @@ public Point3D interpolate(Point3D endValue, double t) {
     }
 
     /**
-     * Returns a hash code value for the point.
-     * @return a hash code value for the point.
+     * Indicates whether some other object is "equal to" this one.
+     * Two instances of {@code Point3D} are equal if the return values of their
+     * {@code getX}, {@code getY}, and {@code getZ} methods are equal.
+     *
+     * @param {@code obj} the reference object with which to compare
```

Comment:
Don't use `@code` for the parameter declaration, they are treated automatically by the javadoc tool.

Reply:
Don't use it for `obj` you mean?